raiden/utils/testnet.py

```diff
@@ -0,0 +1,71 @@
+from typing import TYPE_CHECKING
+
+from eth_utils import to_checksum_address
+
+from raiden.exceptions import RaidenError, TransactionNotSuccessful
+from raiden.network.rpc.transactions import check_transaction_threw
+from raiden.utils.typing import Any, List, TokenAddress, TransactionHash
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import
+    from raiden.network.rpc.client import JSONRPCClient
+    from raiden.network.rpc.smartcontract_proxy import ContractProxy
```

Comment:
Is there a circular import issue that made you import these under `TYPE_CHECKING? if not, please move them from the `TYPE_CHECKING` block to top-level imports and unquote the types.